ImperatorToCK3Tests/CK3WorldTests/Province/CK3ProvinceTests.cpp

```diff
@@ -72,12 +72,12 @@ TEST(ImperatorWorld_ProvinceTests, setHoldingLogicWorks)
 	std::stringstream input5{ " = { province_rank=settlement fort=yes }" };
 	std::stringstream input6{ " = { province_rank=settlement }" };
 
-	auto impProvince = std::make_shared<Imperator::Province>(input, 42);
-	auto impProvince2 = std::make_shared<Imperator::Province>(input2, 43);
-	auto impProvince3 = std::make_shared<Imperator::Province>(input3, 44);
-	auto impProvince4 = std::make_shared<Imperator::Province>(input4, 45);
-	auto impProvince5 = std::make_shared<Imperator::Province>(input5, 46);
-	auto impProvince6 = std::make_shared<Imperator::Province>(input6, 47);
+	std::shared_ptr<Imperator::Province> impProvince = Imperator::Province::Factory().getProvince(input, 42);
```

Comment:
As elsewhere, if using the factory multiple times, it might be worth saving it instead of constructing a new one each time.

Reply:
That's right, I just replaced without thinking.